manuscript/process.md

```diff
@@ -79,4 +79,70 @@ With this more robust goal in mind we're more likely to communicate,
 not just the technique,
 but also its importance and trade-offs.
 
+## 2. Determine the Audience
+
+Once we understand the goal of the document,
+we have enough information to start considering our audience.
+Our first question, though, takes a step back from the goal.
+
+### Who will be reading this?
+
+Often the readership of documents is wider than we would initially expect.
+A framework's website will be read by developers, of course,
+but perhaps also:
+
+* Architects evaluating technology choices
+* Managers trying to keep abreast of what their team is using
+* Technical recruiters trying to understand the skillset they are hiring for
+* Designers trying to understand technical constraints on their designs
```

Comment:
Oh I think *on* is correct, since the design is not the origin of the constraints.